app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Post;
+use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\View\View;
+
+class PostController extends Controller
+{
+    /**
+     * Display the index blog page.
+     *
+     * @return View The posts/index.blade.php view
+     */
+    public function index(): View
+    {
+        return view('posts.index', [
+            'posts' => Post::latest()->simplePaginate(10)
+        ]);
+    }
+
+    /**
+     * Create a new blog post
+     * @return View The posts/create.blade.php view
+     */
+    public function create(): View
+    {
+        return view('posts.create');
+    }
+
+    /**
+     * Edit a blog post
+     * @return View The posts/edit.blade.php view
+     */
+    public function edit($slug): View
```

Comment:
Route model binding can be used either when another column is used for its primary key than $id